core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -15,6 +15,13 @@
 
     <div ng-repeat="question in data.questionList.getQuestions() track by $index">
       <anchor-header header-id="question.getId()" header-text="Question <[$index +1]>"></anchor-header>
+      <anchor-header header-id="getSubfieldId(question,'Multiple Choice')"
+                     header-text="<[getQuestionTypeLabel(question.getInteractionId())]> &#x25BC;" ng-click="chooseQuestionType = true; setModalVisibility(true);">
```

Comment:
Place the setter for `chooseQuestionType` behind a function call. Also with `setModalVisibility(true)` should be done with a more abstract function (ie. `setModalVisible`)